os_stub/rnglib_std/rng.c

```diff
@@ -20,17 +28,47 @@
 **/
 boolean get_random_number_64(OUT uint64 *rand_data)
 {
-	uint8 *ptr;
-
-	ptr = (uint8 *)rand_data;
-	ptr[0] = (uint8)rand();
-	ptr[1] = (uint8)rand();
-	ptr[2] = (uint8)rand();
-	ptr[3] = (uint8)rand();
-	ptr[4] = (uint8)rand();
-	ptr[5] = (uint8)rand();
-	ptr[6] = (uint8)rand();
-	ptr[7] = (uint8)rand();
+#ifdef _MSC_VER
+	HCRYPTPROV hCryptProv;
+
+	if(!CryptAcquireContext(
+		&hCryptProv,
+		NULL,
+		NULL,
+		PROV_RSA_FULL,
+		0)) {
+		return FALSE;
+	}
+
+	if(!CryptGenRandom(
```

Comment:
It looks like this API is deprecated. https://docs.microsoft.com/en-us/windows/win32/api/wincrypt/nf-wincrypt-cryptgenrandom. Perhaps we should use the Crypto Next Gen APIs?

Reply:
done